k8s/namespaces/wa/wa-task-monitor/prod.yaml

```diff
@@ -8,3 +8,4 @@ spec:
       environment:
         OPEN_ID_IDAM_URL: "https://hmcts-access.service.gov.uk"
         CONFIGURATION_CAMUNDA_MAX_RESULTS: "100"
+        PENDING_TERMINATION_CAMUNDA_MAX_RESULTS: "100"
```

Comment:
We _may_ want to start with a lower number in prod just so we can check it worked?